pyramid_hypernova/request.py

```diff
@@ -13,8 +13,11 @@ def create_jobs_payload(jobs):
 
 
 class HypernovaQueryError(Exception):
-    def __init__(self, child_error):
+    # error_data (obj, optional): { name, message, stack }
```

Comment:
i'm guessing since this package isn't mypy typed that this is mainly for human consumption?

we should probably use the [type comments](https://peps.python.org/pep-0484/#type-comments) syntax and/or a proper docstring if so

additionally, we should consider using a more strict type for this like a namedtuple or a dataclass so that we're not passing around loose objects that need this kind of human verification to begin with